web/src/routes/(user)/people/+page.svelte

```diff
@@ -159,10 +159,19 @@
       return;
     }
     try {
-      await api.personApi.mergePerson({
+      const { data } = await api.personApi.mergePerson({
         id: personMerge2.id,
         mergePersonDto: { ids: [personToMerge.id] },
       });
+      if (data.person.birthDate != personMerge2.birthDate) {
+        for (const person of people) {
+          if (person.id === data.person.id) {
+            person.birthDate = data.person.birthDate;
+            break;
+          }
+        }
```

Comment:
Why is the frontend updating person information (birth date)?

Reply:
Because when you merge one person with his birthDate set to a person without the birthDate set, you want to keep the birthDate

Reply:
Yeah I get the usage but shouldn't that be taken care of by the merge on the server side?

Reply:
Yep fixed